src/docs/src/install/freebsd.rst

```diff
@@ -16,71 +16,93 @@
 Installation on FreeBSD
 =======================
 
-Installation from ports
-=======================
+Installation
+============
 
-.. code-block:: text
+Use `the pre-built binary packages`_ to install CouchDB::
 
-    cd /usr/ports/databases/couchdb
-    make install clean
+    pkg install couchdb3
 
-This will install CouchDB from the ports collection.
+Alternatively, it is possible installing CouchDB from the Ports
+Collection::
+
+    cd /usr/ports/databases/couchdb3
+    make install clean
 
 .. note::
    Be sure to :ref:`create an admin user<config/admins>` before starting
    CouchDB for the first time!
 
-Start script
-------------
+Service Configuration
+=====================
 
-The following options for ``/etc/rc.conf`` or ``/etc/rc.conf.local`` are
-supported by the start script (defaults shown)::
+The port is shipped with a script that integrates CouchDB with
+FreeBSD's `rc.d service framework`_.  The following options for
+``/etc/rc.conf`` or ``/etc/rc.conf.local`` are supported (defaults
+shown)::
 
-    couchdb_enable="NO"
-    couchdb_enablelogs="YES"
-    couchdb_user="couchdb"
+    couchdb3_enable="NO"
+    couchdb3_user="couchdb"
+    couchdb3_erl_flags="-couch_ini /usr/local/libexec/couchdb3/etc/default.ini /usr/local/etc/couchdb3/local.ini"
+    couchdb3_chdir="/var/db/couchdb3"
 
-After enabling the couchdb rc service use the following command to start CouchDB::
+After enabling the ``couchdb3`` service (by setting
+``couchdb3_enable`` to ``"YES"``), use the following command to start
+CouchDB::
 
-    /usr/local/etc/rc.d/couchdb start
+    service couchdb3 start
 
-This script responds to the arguments `start`, `stop`, `status`, `rcvar` etc..
+This script responds to the arguments `start`, `stop`, `status`,
+`rcvar` etc.  If the service is not yet enabled in ``rc.conf``, use
+``onestart`` to start it up ad-hoc.
 
-The start script will also use settings from the following config files:
+The service will also use settings from the following config files:
 
-- /usr/local/etc/couchdb/default.ini
-- /usr/local/etc/couchdb/local.ini
+- ``/usr/local/libexec/couchdb3/etc/default.ini``
```

Comment:
I think these (and other) paths will vary depending on whether this is installed from sources, or via packages/ports.

Either way making this more explicit is a good thing --

`default.ini` should be left read-only, and will be replaced on upgrades and re-installs without warning. Local modifications can go in `local.ini` as usual.

Does that read a bit better?

Reply:
I think the `rc.d` script is shipped only with the version from ports/packages.  Therefore it makes clear that we are not talking about installing the vanilla CouchDB from sources.  That said, I would not worry about the differences in the location of the configuration files for now.

Reply:
I used the sentence about `default.ini` in the text though because I agree that it gives a better understanding of its nature.